plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/utils.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { KeyedFormField } from '@woocommerce/settings';
+
+export interface FieldProps {
+	id: string;
+	errorId: string;
+	label: string | undefined;
+	autoCapitalize: string | undefined;
+	autoComplete: string | undefined;
+	errorMessage: string | undefined;
+	required: boolean | undefined;
+	className: string;
+}
+
+export const createFieldProps = (
```

Comment:
Inline documentation would be great for this

Reply:
Sounds good. I've just documented this function.